src/main/java/com/epam/test_generator/transformers/CustomCaseVersionConverter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.test_generator.transformers;
+
+import com.epam.test_generator.dto.StepDTO;
+import com.epam.test_generator.entities.Step;
+import com.epam.test_generator.entities.Tag;
+import org.dozer.CustomConverter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CustomCaseVersionConverter implements CustomConverter {
+
+    @Override
+    public Object convert(Object existingDestinationFieldValue, Object sourceFieldValue,
```

Comment:
Don't use Object. There are a lot of useful patterns to store different types. For example Either.